### PR TITLE
NN-3152 Whereabouts residential location search page

### PR DESCRIPTION
**Pull request by jsrobertson**
image
image

### CODE REVIEW

File: integration-tests/integration/whereabouts/selectResidentialLocation.spec.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const moment = require('moment')
+
+// copy of shared/getCurrentTimePeriod.js
```
Comment (jsrobertson) [inline integration-tests/integration/whereabouts/selectResidentialLocation.spec.js:3]: Didn't want to include it directly incase it changes and the tests pass incorrectly.

File: backend/shared/getCurrentTimePeriod.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const moment = require('moment')
```
Comment (JamieDraperUK) [inline backend/shared/getCurrentTimePeriod.js:1]: On the off chance you don't know, there is a version of this in backend/utils.js

File: src/ResultsHouseblock/ResultsHouseblockContainer.test.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ describe('ResultsHouseblockContainer', () => {
       expect(wrapper.find('Connect(Page)').prop('title')).toEqual('Houseblock 1 - Leaving')
     })
 
-    it('should have the correct page title when when a sub location is selected', () => {
+    it.skip('should have the correct page title when when a sub location is selected', () => {
```
Comment (JamieDraperUK) [inline src/ResultsHouseblock/ResultsHouseblockContainer.test.js:66]: Was this checked in by mistake?

**Review by JamieDraperUK — DISMISSED** [review]
LGTM - Nice work

**Review by JamieDraperUK — APPROVED** [review]

**Pull request merged**